backend/src/main/java/cloudflight/integra/backend/controller/TestController.java

```diff
@@ -1,15 +1,41 @@
 package cloudflight.integra.backend.controller;
 
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.RestController;
+import cloudflight.integra.backend.model.Proposal;
+import cloudflight.integra.backend.service.ServiceProposal;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
 
 import java.util.List;
 
 @RestController
+@RequestMapping
 public class TestController {
+    private ServiceProposal serviceProposal;
 
-    @GetMapping("/test")
-    public List<String> testEndpoint() {
-        return List.of("hello", "world");
+    @Autowired
+    private void setProposalService(ServiceProposal serviceProposal) {
+        this.serviceProposal = serviceProposal;
+    }
```

Comment:
Can we create a separate Controller for the Proposal Entity? 

```java
@RestController
@RequestMapping("/proposal")
public class ProposalController 
```

All the methods implemented below this point should be moved to the new Controller